### PR TITLE
Adding like% and %like in filter

### PR DESCRIPTION
**Pull request by ElNelyo**
[From this issue](https://github.com/stwe/DatatablesBundle/issues/848)

### CODE REVIEW

File: Datatable/Filter/AbstractFilter.php [hunk 1]
```diff
@@ -130,7 +130,7 @@ public function configureOptions(OptionsResolver $resolver)
         $resolver->setAllowedTypes('placeholder', 'bool');
         $resolver->setAllowedTypes('placeholder_text', array('null', 'string'));
 
-        $resolver->setAllowedValues('search_type', array('like', 'notLike', 'eq', 'neq', 'lt', 'lte', 'gt', 'gte', 'in', 'notIn', 'isNull', 'isNotNull'));
+        $resolver->setAllowedValues('search_type', array('like','%like','like%','notLike', 'eq', 'neq', 'lt', 'lte', 'gt', 'gte', 'in', 'notIn', 'isNull', 'isNotNull'));
```
Comment (Seb33300) [inline Datatable/Filter/AbstractFilter.php]: Missing spaces after `,`

File: Datatable/Options.php [hunk 2]
```diff
@@ -318,7 +318,7 @@ public function configureOptions(OptionsResolver $resolver)
         $resolver->setAllowedTypes('global_search_type', 'string');
 
         $resolver->setAllowedValues('individual_filtering_position', array('head', 'foot', 'both'));
-        $resolver->setAllowedValues('global_search_type', array('like', 'notLike', 'eq', 'neq', 'lt', 'lte', 'gt', 'gte', 'in', 'notIn', 'isNull', 'isNotNull'));
+        $resolver->setAllowedValues('global_search_type', array('like','%like','like%','notLike', 'eq', 'neq', 'lt', 'lte', 'gt', 'gte', 'in', 'notIn', 'isNull', 'isNotNull'));
```
Comment (Seb33300) [inline Datatable/Options.php]: Missing spaces after `,`

File: Datatable/Filter/AbstractFilter.php [hunk 3]
```diff
@@ -400,6 +400,14 @@ protected function getExpression(Composite $expr, QueryBuilder $qb, $searchType,
         }
 
         switch ($searchType) {
+            case 'like%':
```
Comment (Seb33300) [inline Datatable/Filter/AbstractFilter.php]: You should keep the same order as previously.

**Comment by ElNelyo** [comment]
Oh sorry, thanks for the advices @Seb33300

**Comment by ElNelyo** [comment]
Hi @stephanvierkant , can you validate this PR ? 
Have a nice day

**Pull request merged**